Makefile

```diff
@@ -316,7 +316,8 @@ lint: ##@test Run code style checks
 	ALL_CLOJURE_FILES=$(call find_all_clojure_files) && \
 	zprint '{:search-config? true}' -sfc $$ALL_CLOJURE_FILES && \
 	sh scripts/lint-trailing-newline.sh && \
-	node_modules/.bin/prettier --write .
+	node_modules/.bin/prettier --write . && \
+	scripts/lint_translations.clj
```

Comment:
The `lint` target is quite slow, so I think the most valuable checks should be performed earlier. I think this linting step should happen after `clj-kondo` is called. The formatting checks are less valuable and for many unnecessary (usually) because we format automatically in the editor when saving files.